src/TestReports.jl

```diff
@@ -31,9 +31,23 @@ end
 
 export ReportingTestSet, any_problems, report, recordproperty
 
+const TESTREPORTS_VERSION = let # Copied from Documenter.jl
+    project = joinpath(dirname(dirname(pathof(TestReports))), "Project.toml")
+    toml = read(project, String)
+    m = match(r"(*ANYCRLF)^version\s*=\s\"(.*)\"$"m, toml)
```

Comment:
Could probably use the TOML stdlibs here?
Which might be more robust?

Reply:
I'll use `Pkg.TOML` as it wasn't a stdlib in v1.0.5